app/code/Tasks/FeedbackManager/Block/FeedbackAdd.php

```diff
@@ -0,0 +1,57 @@
+<?php
+namespace Tasks\FeedbackManager\Block;
+
+use Magento\Customer\Model\SessionFactory;
+
+class FeedbackAdd extends \Magento\Framework\View\Element\Template
+{
+   
+     /** @var SessionFactory */
+     protected $session;
+
+     /** @var */
+     protected $customerData;
+ 
+     /**
+      * Feedback constructor.
+      * @param Template\Context $context
+      * @param SessionFactory $session
+      * @param array $data
+      */
+ 
+     public function __construct(
+        \Magento\Framework\View\Element\Template\Context $context,
+         SessionFactory $session,
+         array $data = []
+     )
+     {
+         parent::__construct($context, $data);
+         $this->session= $session->create();
+         $this->customerData = ($this->session->isLoggedIn()) ? $this->session->getCustomerData() : null;
+     }
+ 
+     
+     public function getCustomerFirstName()
+     {
+         if($this->customerData) 
+              return $this->customerData->getFirstname();
+         else 
+             return false;
+     }
+ 
+     public function getCustomerLastName()
+     {
+         if($this->customerData) 
+              return $this->customerData->getLastname();
+         else 
+             return false;
+     }
+ 
+     public function getCustomerEmail()
```

Comment:
@gayathri-aravind Use guard clause and avoid else

Reply:
Used guard clause and the corresponding commit is 4c36909c6b081c0188054a8d192aa4c2d4a30970.